src/utils/trust/common.trust.data.mapper.ts

```diff
@@ -0,0 +1,18 @@
+import * as Trust from 'model/trust.model';
+import { CommonTrustData } from 'model/trust.page.model';
+
+const mapCommonTrustDataToPage = (
```

Comment:
```
const mapTrustToPage = (
  appData: ApplicationData,
  trustId: string,
): Page.TrustHistoricalBeneficialOwner => {
  const data = getTrustByIdFromApp(appData, trustId);

  return {
    trustId: data.trust_id,
    trustName: data.trust_name,
  };
};
```

please use this one

Reply:
Updated this mapper. However I am considering adding your `getTrustByIdFromApp ` method and relevant code to my branch so the code doesn't have to be revisited in future.